components/Form.js

```diff
@@ -0,0 +1,86 @@
+import Button from "./Button";
+import { collections } from "@/lib/data";
+import styled from "styled-components";
+import { uid } from "uid";
+import { useState } from "react";
+
+const StyledFormContainer = styled.div``;
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  padding: 1em 2em;
+`;
+
+const SmallText = styled.p`
+  font-size: 0.8rem;
+`;
+
+const SubmitMessage = styled.p`
+  text-align: center;
+`;
+
+export default function Form({ onAddFlashCard }) {
+  const [isCardCreatedMessage, setIsCardCreatedMessage] = useState("");
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    setIsCardCreatedMessage("New Card Created.");
+
+    const formElements = event.target.elements;
+
+    const formAnswer = formElements.answer.value;
+    const formQuestion = formElements.question.value;
+    const selectedCollectionTitle = formElements.collections.value;
```

Comment:
Currently, values are extracted manually from `event.target.elements`. Instead, we can use `FormData` to dynamically get form values.

```suggestion
  const formData = new FormData(event.target);
    const data = Object.fromEntries(formData);
    ...
    const newFlashCard = {
      id: uid(),
      collectionId,
      ...data,
      isCorrect: false,
    };
```